src/app/modules/backbone/components/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,35 @@
+.main-nav {
+    @apply flex flex-row items-center content-between;
+    @apply w-full p-2;
+
+    @apply md:p-5 md:flex-col md:h-screen;
+}
+
```

Comment:
I have also added a new item to the guidelines to split tailwind classes into multiple lines according to their intended behaviour (layout, sizing, etc.). easier to read